exporter/awsxrayexporter/translator/segment.go

```diff
@@ -0,0 +1,367 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package translator
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"math/rand"
+	"reflect"
+	"regexp"
+	"sync"
+	"time"
+
+	resourcepb "github.com/census-instrumentation/opencensus-proto/gen-go/resource/v1"
+	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
+	"github.com/golang/protobuf/ptypes/timestamp"
+	tracetranslator "github.com/open-telemetry/opentelemetry-collector/translator/trace"
+)
+
+// OpenTelemetry Semantic Convention values for general Span attribute names.
+const (
+	ComponentAttribute   = "component"
+	PeerAddressAttribute = "peer.address"
```

Comment:
Same for this, would be best to move all OpenTelemetry semantic convention constants to core repo.

Reply:
Moved to collector